moped-database/migrations/1644473675208_set_sequence_pointers/up.sql

```diff
@@ -0,0 +1,29 @@
+SELECT pg_catalog.setval('public.moped_components_component_id_seq', 59, true);
+SELECT pg_catalog.setval('public.moped_department_department_id_seq', 10, true);
+SELECT pg_catalog.setval('public.moped_entities_entity_id_seq', 36, true);
+SELECT pg_catalog.setval('public.moped_fund_sources_funding_source_id_seq', 25, true);
```

Comment:
when I spin up locally and check the funding source table, I don't see any source with id 25, it goes from 24 Williamson County to 26 None